spec.txt

```diff
@@ -4229,10 +4221,10 @@ put under the list item:
 
 
 It is tempting to think of this in terms of columns:  the continuation
-blocks must be indented at least to the column of the first
-[non-whitespace character] after the list marker. However, that is not quite right.
-The spaces after the list marker determine how much relative indentation
-is needed.  Which column this indentation reaches will depend on
+blocks must be indented at least to the column of the first character other than
+a space or tab after the list marker.  However, that is not quite right.
```

Comment:
```suggestion
a space or tab after the list marker. However, that is not quite right.
```

Reply:
Reason for this one is that the spec seems to mostly use two spaces!